recipe/meta.yaml

```diff
@@ -186,8 +186,8 @@ outputs:
   - name: libtorch
   - name: pytorch
     build:
-      string: cuda{{ cuda_compiler_version | replace('.', '') }}_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
-      string: cpu_{{ blas_impl }}_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                 # [cuda_compiler_version == "None"]
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}_{{ blas_impl }}_h{{ PKG_HASH }}_{{ build }}  # [cuda_compiler_version != "None"]
+      string: cpu_{{ blas_impl }}_h{{ PKG_HASH }}_{{ build }}                                                 # [cuda_compiler_version == "None"]
```

Comment:
Please don't remove the `py {{ CONDA_PY }}` from `pytorch`. Here's what we have on main:
https://github.com/conda-forge/pytorch-cpu-feedstock/blob/f3f31a9ee57ab64d53cbc171502ed477e3ac76df/recipe/meta.yaml#L242-L247

Reply:
Oh, did I mess up copying? Sorry.